src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx

```diff
@@ -35,7 +35,7 @@ Requirements and recommendations:
 * Supports SCIM 2.0 standard.
 * [User model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models)-related requirements:
   * This feature requires you to be on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) and creates users on that model. If you're on our [original user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) (or otherwise can't implement this feature), talk to your New Relic account representative.
-  * Configuration requires that a user have the [**Authentication domain manager** and the **Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#standard-roles) (users in the [default group **Admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups) have these).
+  * Configuration requires that a user have the [**Authentication domain manager** and the **Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#roles) (users in the [default group **Admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#default-groups) have these).
```

Comment:
@zuluecho9, the link for **Authentication domain manager** and the **Organization manager** role failed to load locally: http://localhost:8000/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#roles